src/renderer/components/+preferences/preferences-navigation/extension-tab-preferences-navigation-item.injectable.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { computed } from "mobx";
+import { map } from "lodash/fp";
+
+import routeIsActiveInjectable from "../../../routes/route-is-active.injectable";
+import { preferenceNavigationItemInjectionToken } from "./preference-navigation-items.injectable";
+
+import type { LensRendererExtension } from "../../../../extensions/lens-renderer-extension";
+import { extensionRegistratorInjectionToken } from "../../../../extensions/extension-loader/extension-registrator-injection-token";
+import { pipeline } from "@ogre-tools/fp";
+import extensionPreferencesRouteInjectable from "../../../../common/front-end-routing/routes/preferences/extension/extension-preferences-route.injectable";
+import currentPathParametersInjectable from "../../../routes/current-path-parameters.injectable";
+import navigateToExtensionPreferencesInjectable from "../../../../common/front-end-routing/routes/preferences/extension/navigate-to-extension-preferences.injectable";
+import type { LensExtension } from "../../../../extensions/lens-extension";
+
+const extensionSpecificTabNavigationItemRegistratorInjectable = getInjectable({
+  id: "extension-specific-tab-preferences-navigation-items",
+
+  instantiate: (di) => {
+    return (ext: LensExtension) => {
+      const extension = ext as LensRendererExtension;
+      const navigateToExtensionPreferences = di.inject(
+        navigateToExtensionPreferencesInjectable,
+      );
+      const route = di.inject(extensionPreferencesRouteInjectable);
+      const routeIsActive = di.inject(routeIsActiveInjectable, route);
+      const pathParameters = di.inject(currentPathParametersInjectable);
+
+      const injectables = pipeline(
+        extension.appPreferenceTabs,
+
+        map((tab) => {
+          const id = `extension-${extension.sanitizedExtensionId}-nav-item-${tab.id}`;
```

Comment:
For now, need to know stuff. In line 19. there is second argument which is `extensionInstallationCount` Let's add that to the `id` in line 36. This is to allow enabling, disabling and re-enabling extension. 

This is something that we'll get rid of soon being absurd technical complexity. :)

Reply:
Yes, and my additional question is do we really need such long ids? 
```
const id = `extension-${extension.sanitizedExtensionId}-${extensionInstallationCount}-nav-item-${tab.id}`;
```
They really stand out from the "native" ids on the preferences sidebar list.

Reply:
https://github.com/lensapp/lens/pull/5662

This PR tackles the thing I mentioned in earlier comment. 

Value of ID doesn't really matter as long as it's unique.